src/components/FlickityCarousel.js

```diff
@@ -21,7 +21,7 @@ class FlickityCarousel extends Component {
     const { children, changeCaption, ApplyCarousel } = this.props
     if (ApplyCarousel) {
       const captions = r.map(r.view(r.lensPath(["props", "caption"])), children)
-      this.flkty.on("dragEnd", () => {
+      this.flkty.on("change", () => {
```

Comment:
Can we add another dragEnd event listener here that calls the same anonymous callback?